src/Adyen/Util/Util.php

```diff
@@ -197,4 +197,22 @@ public static function isValidNotificationHMAC($params, $hmacKey)
 
         return $expectedSign == $merchantSign;
     }
+
+    /**
+     * Get the Customer Area PSP Search URL with a preset PSP Reference
+     *
+     * @param string $pspReference
+     * @param string $checkoutEnvironment
+     * @return string
+     */
+    public static function getPspReferenceSearchUrl($pspReference, $checkoutEnvironment)
```

Comment:
shall we move this to it's own utility class so we don't create a massive god class ?